src/datasim_ui/db.cljs

```diff
@@ -33,6 +33,11 @@
                                :options/username
                                :options/password]))
 
+(s/def :validation/visible boolean?)
+(s/def :validation/data some?)
```

Comment:
if this spec has a gen, it produces hilarious (but truthy) values. It probably doesn't though

Reply:
fully speced